src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <nav class="nav">
+      <ul class="stars--0">
```

Comment:
```suggestion
      <ul class="stars stars--0">
```

Reply:
fix all similar cases

Reply:
Thanks for your review! I fixed it. Please take a look